.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Make sure you are running the latest version before reporting an issue.
+
+**Component/platform:**
```

Comment:
i think it may be beneficial to "hint" at things we want them to mention (i.e. android auto version, phone, OS, configuration, etc)

Reply:
Good point, i'll glance over issues and look at the questions usually missed.

Reply:
I've added a few extra bits of information and a example row in the table